examples/eql_harmonic.py

```diff
@@ -0,0 +1,64 @@
+"""
+Grid total-field magnetic data from Rio de Janeiro
+==================================================
+
+Most potential field surveys gather data on several scatter points that might be at
+different heights. For a great number of applications we may need to interpolate these
+data points into a regular grid. This can be done through an Equivalent Layer
+interpolator. We will use :class:`harmonica.EQLHarmonic` to generate a set of point
+sources beneath the observation points that predicts the observed data. Then these point
+sources will be used to interpolate the data values into a regular grid at a constant
+height.
+"""
+import harmonica as hm
+import verde as vd
+import numpy as np
+import pyproj
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+from sklearn.model_selection import ShuffleSplit
+
+
+# Fetch magnetic anomaly data from Rio de Janeiro
+data = hm.datasets.fetch_rio_magnetic()
```

Comment:
Get a smaller region on the magnetic anomaly data instead of downsampling it. Try to catch a dipole.